lib/project_types/script/messages/messages.rb

```diff
@@ -24,7 +24,7 @@ module Messages
           oauth_help: "Wait a few minutes and try again.",
 
           invalid_context_cause: "Your .shopify-cli.yml is formatted incorrectly. It's missing values for "\
-                                 "extension_point_type or script_name.",
+                                 "extension_point_type or title.",
```

Comment:
You don't need to change anything, this is just a general comment: it would be nice if this error message was more precise to state the exact field that is missing, not two where one or more is missing.